bin/nova-hypervisor-usage

```diff
@@ -184,6 +186,8 @@ def filter_by_spec(flavor, hypervisors):
             pass
     return valid_hypervisors
 
+def compute_pci_slots_node_capability(max_vms, curr_vms):
+    return max_vms - curr_vms
```

Comment:
This seems unnecessary: a function that returns the difference of its argument does not help readability.